auth-api/src/auth_api/services/affiliation_invitation.py

```diff
@@ -306,10 +306,13 @@ def create_affiliation_invitation(
         affiliation_invitation.token = confirmation_token
         affiliation_invitation.login_source = mandatory_login_source
         affiliation_invitation.save()
-
+        if business["business"]["legalType"] not in ["SP", "GP"]:
+            business_name = business["business"]["legalName"]
+        else:
+            business_name = business["business"]["alternateNames"]["name"]
```

Comment:
I think this is an array? and there's some logic to grab the correct one?

![image](https://github.com/user-attachments/assets/312a0d32-8198-40ed-8c28-c8fcbde6aa8c)